src/main/webapp/admin_dashboard_script.js

```diff
@@ -0,0 +1,37 @@
+google.charts.load('current', {'packages':['line']});
+google.charts.setOnLoadCallback(drawChart);
+
+document.getElementsByTagName("body").addEventListener("onresize", drawChart);
```

Comment:
I get this error locally:

```
Uncaught TypeError: document.getElementsByTagName(...).addEventListener is not a function
    at admin_dashboard_script.js:4
```

Note that `getElementsByTagName` returns an `HTMLCollection`, not an `HTMLElement`, so `addEventListener` isn't valid. Since there's only one `body`, though, you could use `document.querySelector` to get the single element back, then add the event listener.